pkg/securityscan/scanHandler.go

```diff
@@ -142,6 +158,20 @@ func (c *Controller) handleClusterScans(ctx context.Context) error {
 	return nil
 }
 
+func (c *Controller) isScanPresent(scanName string) (bool, error) {
+	scans := c.cisFactory.Cis().V1().ClusterScan()
```

Comment:
Can this be initialized on the Controller?

Reply:
ok, done